.eslintrc.js

```diff
@@ -351,7 +351,7 @@ module.exports = {
                 from: [
                   '(src|x-pack)/plugins/**/(public|server)/**/*',
                   '!(src|x-pack)/plugins/**/(public|server)/mocks/index.{js,mjs,ts}',
-                  '!(src|x-pack)/plugins/**/(public|server)/(index|mocks).{js,mjs,ts,tsx}',
+                  '!(src|x-pack)/plugins/**/(public|server)/(index|mocks|test_utils).{js,mjs,ts,tsx}',
```

Comment:
we can make it a part of mock, but it's not entirely the same. open to suggestions.